src/game_battlealgorithm.cpp

```diff
@@ -667,7 +667,7 @@ void Game_BattleAlgorithm::AlgorithmBase::Apply() {
 	if (GetAffectedAttack() != -1) {
 		int atk = GetAffectedAttack();
 		target->ChangeAtkModifier(IsPositive() ? atk : -atk);
-		if (IsAbsorb()) {
+		if (IsAbsorb() && Player::IsRPG2k3()) {
```

Comment:
Absorb stats not supported in 2k3 v1.0.3.0

Change this to `IsAbsorb() && Player::IsRPG2k3Updated()`